core/common/src/main/java/alluxio/worker/dora/DoraWorker.java

```diff
@@ -37,9 +37,14 @@ public interface DoraWorker extends DataWorker, SessionCleanable {
 
   /**
    * List status from Under File System.
-   * @param path
-   * @param options
-   * @return list of status
+   *
+   * Please refer to UnderFileSystem.listStatus().
+   *
+   * @param path the path of a dir or file
+   * @param options the option for listStatus()
+   * @return An array with the statuses of the files and directories in the directory denoted by
+   *         this abstract pathname. The array will be empty if the directory is empty. Returns
+   *         {@code null} if this abstract pathname does not denote a directory.
    * @throws IOException
    */
   UfsStatus[] listStatus(String path, ListOptions options) throws IOException;
```

Comment:
Annotate with `@Nullable` if this can return `null`, so that the IDE can help check `null`s.

Reply:
Fixed.